src/app/components/Footer.tsx

```diff
@@ -0,0 +1,343 @@
+//                   {links.map(({ name, href }) => (
+//                     <li key={name}>
+//                       <Link href={href} className="hover:text-[#F2B263]">
+//                         {name}
+//                       </Link>
+//                     </li>
+//                   ))}
+//                 </ul>
+//               </div>
+//             ))}
+//             {/* Socials */}
+//             <div>
+//               <h3 className="text-sm sm:text-lg font-bold mb-4">Socials</h3>
+//               <div className="grid grid-cols-2 gap-0.1 justify-items-center mt-1">
+//                 {socialLinks.map(({ name, url, icon }) => (
+//                   <Link
+//                     key={name}
+//                     href={url}
+//                     target="_blank"
+//                     className="hover:text-[#F2B263] p-2"
+//                   >
+//                     <Icon icon={icon} width={25} height={25} />
+//                   </Link>
+//                 ))}
+//               </div>
+//             </div>
+//           </div>
+
+//           {/* Logo Centered at Bottom */}
+//           <div className="mt-2 sm:mt-4 md:mt-6 mb-4">
+//             <Image
+//               src="/images/LOGO.png"
+//               alt="Currenci Logo"
+//               className="w-32 h-auto mx-auto"
+//               width={150}
+//               height={150}
+//             />
+//           </div>
+//         </div>
+//       </div>
+
+//       {/* Footer Bottom */}
+//       <div className="bg-[#1E3432] text-center text-[#a6a6a6] text-sm py-4">
+//         <div className="w-[80%] h-[2px] bg-[#3A4A47] mx-auto mb-2" />
+//         <p>© 2024 Currenci | All Rights Reserved | Developers</p>
+//       </div>
+//     </footer>
+//   );
+// }
+import Link from "next/link";
+import Image from "next/image";
+import { Icon } from "@iconify/react";
+
+const sections = [
+  {
+    title: "About Us",
+    links: [
+      { name: "About", href: "/about" },
+      { name: "Projects", href: "/projects" },
+      { name: "Privacy Policy", href: "/privacy-policy" },
+      { name: "Contact Us", href: "/contact" },
+    ],
+  },
+  {
+    title: "Fees",
+    links: [
+      { name: "Plans", href: "/plans" },
+      { name: "Why Us", href: "/why-us" },
+      { name: "FAQs", href: "/faqs" },
+    ],
+  },
+  {
+    title: "Testimonials",
+    links: [
+      { name: "Why Trust Us?", href: "/why-trust-us" },
+      { name: "Our Clients", href: "/our-clients" },
+    ],
+  },
+];
+
+const socialLinks = [
+  { name: "Facebook", url: "https://facebook.com", icon: "mdi:facebook" },
+  { name: "Instagram", url: "https://instagram.com", icon: "mdi:instagram" },
+  { name: "Whatsapp", url: "https://whatsapp.com", icon: "mdi:whatsapp" },
+  { name: "Twitter", url: "https://twitter.com", icon: "mdi:twitter" },
+];
+
+export default function Footer() {
+  return (
+    <footer className="bg-[#1E3432] absolute bottom-11 text-white w-full md:h-[40vh] h-[60vh]">
```

Comment:
It doesn't need an absolute position. The component itself is placed in the bottom(last in order). It will cover the actual content otherwise(try it in local)